scripts/Install-AllSoftware.ps1

```diff
@@ -0,0 +1,24 @@
+# 1. Make sure the Microsoft App Installer is installed:
+#    https://www.microsoft.com/en-us/p/app-installer/9nblggh4nns1
+# 2. Run this script as administrator. If running scripts is blocked, you can temporarily unblock them by running 
+	Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process
```

Comment:
Microsoft needs to sign this script then we can rely on default RemoteSigned behavior and delete step 2 here.

Reply:
Makes sense! Omitted the verbatim in my comments and will look more into this.

Reply:
(If the other developer scripts are unsigned, then disregard.)

Reply:
The other ps scripts are unsigned in this proj so I'll keep this one as unsigned too, but it's definitely a good thing to look into later. Will probably create an issue for it soon.